### PR TITLE
US17295 Jekyll Updates

### PR DESCRIPTION
**Pull request by tcmacdonald**
## Problem
As we've developed more Jekyll projects, we've got a bunch of duplication across codebases. 

## Solution
This PR removes the duplicate stuff and replaces it with an updated version of `jekyll-crds` which has all the things. 

### Related PRs
crdschurch/jekyll-crds#13
crdschurch/crds-media#654

### Notes 
This PR is currently pointed at a branch of jekyll-crds, once merged and released, I'll update this PR to reflect the version number.

### CODE REVIEW

**Comment by tcmacdonald** [comment]
Deploy preview for *int-crds-net* processing.

Building with commit 2b70323dfc24bef5e1f4858ee38805404cfc30ad

https://app.netlify.com/sites/int-crds-net/deploys/5ced326fe92a92000a8af4bd

**Comment by tcmacdonald** [comment]
Deploy preview for *int-crds-net* processing.

Building with commit 2b70323dfc24bef5e1f4858ee38805404cfc30ad

https://app.netlify.com/sites/int-crds-net/deploys/5ced326fe92a92000a8af4c0

File: Gemfile [hunk 1]
```diff
@@ -13,7 +13,7 @@ group :jekyll_plugins do
   gem 'jekyll-asset-pipeline', git: 'https://github.com/crdschurch/jekyll-asset-pipeline', tag: '0.0.2'
   # gem 'jekyll-asset-pipeline', path: File.expand_path('../jekyll-asset-pipeline', __dir__)
   gem "jekyll-contentful", git: 'https://github.com/crdschurch/jekyll-contentful.git', tag: '1.2.1'
-  gem "jekyll-crds", git: 'https://github.com/crdschurch/jekyll-crds.git', tag: '0.0.4'
+  gem "jekyll-crds", git: 'https://github.com/crdschurch/jekyll-crds.git', branch: 'feature/US17295-jekyll-updates'
```
Comment (j-brammer) [inline Gemfile]: I will merge jekyll-crds before this goes in. Can you reference the proper version instead of a local branch here?

**Comment by tcmacdonald** [comment]
@j-brammer This has been updated to reflect the new version of `jekyll-crds`. 

crdschurch/crds-media#654 too. Thanks

File: Gemfile [hunk 2]
```diff
@@ -13,7 +13,7 @@ group :jekyll_plugins do
   gem 'jekyll-asset-pipeline', git: 'https://github.com/crdschurch/jekyll-asset-pipeline', tag: '0.0.2'
   # gem 'jekyll-asset-pipeline', path: File.expand_path('../jekyll-asset-pipeline', __dir__)
   gem "jekyll-contentful", git: 'https://github.com/crdschurch/jekyll-contentful.git', tag: '1.2.1'
```
Comment (j-brammer) [inline Gemfile]: same issue as crds-media, doesn't this need to be 1.3.0?

Comment (tcmacdonald) [reply Gemfile]: @j-brammer I didn't update Jekyll Contentful for these projects yet. I figured we could do that separately if/when you're ready to do so. Its not critical to the work I'm doing for briantome.com

Comment (j-brammer) [reply Gemfile]: Alrighty

**Pull request merged**